.tekton/pipeline-service-ckcp-testing.yaml

```diff
@@ -0,0 +1,295 @@
+    pipelinesascode.tekton.dev/task: "[git-clone]"
+    pipelinesascode.tekton.dev/max-keep-runs: "5"
+spec:
+  params:
+    - name: repo_url
+      value: "{{ repo_url }}"
+    - name: revision
+      value: "{{ revision }}"
+    - name: namespace
+      value: "ci-clusters"
+  timeouts:
+    pipeline: "1h0m0s"
+    finally: "0h20m0s"
+  pipelineSpec:
+    params:
+      - name: repo_url
+      - name: revision
+      - name: namespace
+    workspaces:
+      - name: source
+      - name: kubeconfig-dir
+      - name: hypershift-cluster-template
+      - name: shared-workspace
+    tasks:
+      - name: produce-cluster-name
+        taskSpec:
+          results:
+            - name: cluster-name
+              description: Openshift cluster name
+          steps:
+            - name: cluster-name
+              image: registry.access.redhat.com/ubi8/openssl:8.6-32
+              script: |
+                #!/usr/bin/env bash
+                CLUSTER_NAME="ci-$( openssl rand -hex 5 )"
+                echo -n "$CLUSTER_NAME" | tee $(results.cluster-name.path)
+      - name: deploy-cluster
+        runAfter:
+          - "produce-cluster-name"
+        params:
+          - name: image
+            value: "quay.io/openshift-release-dev/ocp-release:4.11.13-x86_64"
+          - name: region
+            value: "us-east-1"
+          - name: cluster-name
+            value: "$(tasks.produce-cluster-name.results.cluster-name)"
+          - name: namespace
+            value: "$(params.namespace)"
+        workspaces:
+          - name: kubeconfig-dir
+            workspace: kubeconfig-dir
+          - name: hypershift-cluster-template
+            workspace: hypershift-cluster-template
+          - name: output
+            workspace: shared-workspace
+        taskSpec:
+          params:
+            - name: region
+            - name: cluster-name
+            - name: image
+            - name: namespace
+          results:
+            - name: kubeconfig
+              description: the kubeconfig content of the new OCP cluster
+          workspaces:
+            - name: kubeconfig-dir
+            - name: hypershift-cluster-template
+            - name: output
+              description: The kubeconfig of new cluster will be stored onto the volume backing this Workspace
+          kind: ClusterTask
+          steps:
+            - name: generate-hypershift-deployment
+              image: ghcr.io/sharkusmanch/envsubst:1.2.0
+              env:
+                - name: REGION
+                  value: "$(params.region)"
+                - name: CLUSTER_NAME
+                  value: "$(params.cluster-name)"
+                - name: IMAGE
+                  value: "$(params.image)"
+              script: |
+                envsubst < $(workspaces.hypershift-cluster-template.path)/cluster-template.yaml | tee $(workspaces.output.path)/HypershiftDeployment.yaml
+            - name: deploy-cluster
+              image: quay.io/xinjiang/kubectl:1.25
```

Comment:
Change this to the official image. If this is a custom image you've built, then please upload this to quay.io/redhat-pipeline-service organization.

Reply:
fixed, same as above